contracts/src/token/erc721/extensions/consecutive.rs

```diff
@@ -168,7 +168,7 @@ impl IErc721 for Erc721Consecutive {
             from,
             to,
             token_id,
-            &data,
+            data,
```

Comment:
This is more a question than a comment. I'm curious what happened here. I see that `IERC721Receiver` always took bytes as a parameter, but we were sending the borrowed variable through the codebase.

Reply:
Since `Bytes` type just wraps Vec<u8>, that is referential type by itself. We are not copying underlying bytes itself passing this type by value. I would say it is more simple to have it without reference if we can do it